src/zui/components/ZUILink/index.tsx

```diff
@@ -5,6 +5,11 @@ import { FC } from 'react';
 import { ZUIMedium, ZUISmall } from '../types';
 
 type ZUILinkProps = {
+  /**
+   * If true, only show underline when hovering
+   */
+  hoverUnderline?: boolean;
+
```

Comment:
This is a change to the ZUI design system. I just want to double check with @sefsh that this is an intended option for links in ZUI?